<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinBeatifyPhoto</class>
 <widget class="QWidget" name="WinBeatifyPhoto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1191</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="PbFFtHighFilter">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>670</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>FFTHighFilter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbSaveFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Save File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbFuzzy">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>490</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Fuzzy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbLast">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Last</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbGaussian">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>550</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Gaussian</string>
   </property>
  </widget>
  <widget class="QLabel" name="LbOutputImage">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>50</y>
     <width>541</width>
     <height>421</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="midLineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>output image</string>
   </property>
  </widget>
  <widget class="QLabel" name="LbInputImage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>561</width>
     <height>421</height>
    </rect>
   </property>
   <property name="text">
    <string>input image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbOpenFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </widget>
  <widget class="QSlider" name="VsBrightness">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>490</y>
     <width>16</width>
     <height>251</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="VsSaturation">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>490</y>
     <width>16</width>
     <height>251</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="VsConstration">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>490</y>
     <width>16</width>
     <height>251</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="PbBiarization">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>490</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Binarization</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbEquration_k">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>490</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>unused</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbGrayScale">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>610</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Grayscale</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbYMinior">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>550</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>minior_Y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbEquration">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>730</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Equration</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbNext">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>670</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbBackToMain">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>730</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>BackToMain</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbXMinior">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>490</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>minior_X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbCorrosion">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>550</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Corrosion</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbFFtBandFilter">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>730</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>FFTBandFilter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbOilPaint">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>610</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>OilPaint</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbRatation">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>730</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ratation</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SbBinarization">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>490</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="SbCorrosion">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>550</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PbExpand">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>610</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>expand</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SbExpand">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>610</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PbShapen">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>670</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>sharpen</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SbSharpen">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>670</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PbConfirm">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>490</y>
     <width>100</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SbRatation">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>730</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>740</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>BT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>740</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ST</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>740</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>CT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbEquration_9">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>550</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>unused</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbFFtLowFilter">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>610</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>FFTLowFilter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbLaplaceShape">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>730</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>LaplaceShape</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbClassical">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>670</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Classical</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbSketch">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>670</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Sketch</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Source Photo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>10</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Beatify Photo</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="TbHint">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>780</y>
     <width>1101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>780</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hint:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbResetAll">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>590</y>
     <width>100</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>ResetAll</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SbFuzzy">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>490</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="SbGaussian">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>550</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="SbFFtBandFilter1">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>730</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="SbOilPaint">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>610</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="SbSketch">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>670</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="SbFFtBandFilter0">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>730</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="SbFFtHighFilter">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>670</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="SbGaussian_2">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>550</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="SbFFtLowFilter">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>610</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="SbFuzzy_2">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>490</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PbWinAiPhoto">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>680</y>
     <width>100</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>GotoWinAi</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
